schemas/produktside/produktsideGeneralText.ts

```diff
@@ -0,0 +1,29 @@
+export const produktsideGeneralText = {
+  type: "document",
+  name: "produktsideGeneralText",
+  title: "Generelle tekster",
```

Comment:
Jeg tenker det kunne vært bedre med `Produktside generelle tekster` for å tydelig gjøre at det er produktside det tilhører til. 

f.eks her blir det lettere å skille mellom de forskjellige generelle tekster.
<img width="1040" alt="Screenshot 2023-01-25 at 15 11 47" src="https://user-images.githubusercontent.com/110396878/214585719-a3fd8b4c-a7d6-4820-95cc-8a46eb04c89c.png">

Reply:
Sorry, så ikke @hanneolsen  har allerede kommentert på akkurat det samme. My bad 😭

Reply:
Great minds think alike! 😃

Reply:
Det er også veldig bra at du tok med et skjermbilde @nattaphongklinjan. Det begynner å bli mye i Sanity og det kan fort bli uoversiktlig. Skjermbildet ditt illustrerer dette veldig godt.